tests/Unit/MeilisearchEngineTest.php

```diff
@@ -117,12 +118,12 @@ public function test_search_sends_correct_parameters_to_meilisearch()
         $client = m::mock(Client::class);
         $client->shouldReceive('index')->with('table')->andReturn($index = m::mock(Indexes::class));
         $index->shouldReceive('search')->with('mustang', [
-            'filter' => 'foo=1 AND bar=2',
+            'filters' => 'foo=1 AND bar=2',
```

Comment:
It is supposed to be `"filter"` 🤔 

https://github.com/meilisearch/meilisearch-php/blob/19351d7ddff1cf66fe4db33bfdde55e601faa554/tests/Endpoints/SearchTest.php#L254

Reply:
There are more occurrences of this, if I'm correct.